mix.lock

```diff
@@ -1,4 +1,5 @@
 %{
+  "bunt": {:hex, :bunt, "0.2.1", "e2d4792f7bc0ced7583ab54922808919518d0e57ee162901a16a1b6664ef3b14", [:mix], [], "hexpm", "a330bfb4245239787b15005e66ae6845c9cd524a288f0d141c148b02603777a5"},
```

Comment:
Can you explain why Bunt is needed? Looks like an artifact to me.

Reply:
credo depends on bunt, so it got included.

https://github.com/rrrene/credo/blob/master/mix.exs#L151